x/rollup/types/expected_keepers.go

```diff
@@ -10,12 +10,14 @@ import (
 type BankKeeper interface {
 	SendCoinsFromModuleToAccount(ctx context.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
 	SendCoinsFromAccountToModule(ctx context.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
+	SendCoinsFromModuleToModule(ctx context.Context, senderModule, recipientModule string, amt sdk.Coins) error
 
 	MintCoins(ctx context.Context, name string, amt sdk.Coins) error
 	BurnCoins(ctx context.Context, moduleName string, amt sdk.Coins) error
+
+	GetBalance(ctx context.Context, addr sdk.AccAddress, denom string) sdk.Coin
 }
 
 type AccountKeeper interface {
-	NewAccountWithAddress(context.Context, sdk.AccAddress) sdk.AccountI
-	SetAccount(context.Context, sdk.AccountI)
+	GetModuleAddress(moduleName string) sdk.AccAddress
```

Comment:
Woops, was this an unused type before? I might have forgotten to remove it in a previous PR.

Reply:
I didn't look too much into when they were added and if we were previously using them, however they weren't being used on main so I just decided to remove them.